web/client-api/src/main/java/io/deephaven/web/client/api/JsTable.java

```diff
@@ -920,6 +920,11 @@ public Promise<JsTable> naturalJoin(JsTable rightTable, JsArray<String> columnsT
                 .then(state -> Promise.resolve(new JsTable(workerConnection, state)));
     }
 
+    @JsMethod
+    public Promise<TableMap> byExternal(Object keys, @JsOptional Boolean dropKeys) {
```

Comment:
This doesn't match your PR description:
> The JS Table method byExternal() still exists, but now returns a JsPartitionedTable, but the new method partitionBy() is preferred.
It's still returning a TableMap, and the partitionBy looks like it just throws at this point. More changes not pushed?

Reply:
Right, not yet implemented, I'll get this in too, shouldnt take long to deal with these remaining bits. That will reduce TableMap.java to just a few event names, so that the web UI can still point to that class for names, but use the new (same api, different class) JsPartitionedTable.java.